Header header
float64 temp_min
float64 temp_med
float64 temp_max
int32   temp_max_x
int32   temp_max_y
